tracee-rules/signatures/golang/helpers.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/aquasecurity/tracee/tracee-rules/types"
+)
+
+func GetArgumentByName(event types.TraceeEvent, argName string) (types.TraceeEventArgument, error) {
```

Comment:
As we expect more input sources to be added in the future, I think that the function name should include Tracee in it